tpm2/tpm2.go

```diff
@@ -1156,6 +1155,42 @@ func Certify(rw io.ReadWriter, parentAuth, ownerAuth string, object, signer tpmu
 	return decodeCertify(resp)
 }
 
+func encodeCertifyCreation(objectAuth string, object, signer tpmutil.Handle, qualifyingData, creationHash []byte, ticket *Ticket) ([]byte, error) {
+	handles, err := tpmutil.Pack(signer, object)
+	if err != nil {
+		return nil, err
+	}
+	auth, err := encodeAuthArea(HandlePasswordSession, objectAuth)
+	if err != nil {
+		return nil, err
+	}
+	scheme := tpmtSigScheme{AlgRSASSA, AlgSHA256}
+	tk, err := ticket.Encode()
```

Comment:
I just realized that `tpmutil.Pack(ticket)` should "just work" and return the same as `ticket.Encode`.
You can probably pass ticket to `tpmutil.Pack` below directly